Clockify.Net/ClockifyClient.cs

```diff
@@ -419,6 +419,44 @@ public Task<IRestResponse> DeleteTimeEntryAsync(string workspaceId, string timeE
 
 			return _client.ExecuteGetAsync<List<TimeEntryDtoImpl>>(request);
 		}
```

Comment:
Formating is broken here. It needs some fixes with tabs and brackets.

Reply:
I replaced spaces with tabs